ext/com_dotnet/com_extension.stub.php

```diff
@@ -54,32 +54,31 @@ function variant_set_type(variant $variant, int $type): void {}
 
 function variant_cast(variant $variant, int $type): variant {}
 
-function com_get_active_object(string $progid, ?int $code_page = null): variant {}
+function com_get_active_object(string $prog_id, ?int $code_page = null): variant {}
 
 function com_create_guid(): string|false {}
 
-function com_event_sink(variant $comobject, object $sinkobject, array|string|null $sinkinterface = null): bool {}
+function com_event_sink(variant $variant, object $sink_object, array|string|null $sink_interface = null): bool {}
 
-/** @param com|dotnet|variant|string $comobject */
-function com_print_typeinfo($comobject, ?string $dispinterface = null, bool $wantsink = false): bool {}
+function com_print_typeinfo(variant|string $variant, ?string $disp_interface = null, bool $want_sink = false): bool {}
```

Comment:
The convention we have been following is to write everything non-abbreviated, except:
- most obvious things (e.g. arg, param, num)
- if the API embraces an abbreviation too much (stmt - duh, I needed a few tries to correctly spell this)

So I'd prefer something like `$dispatch_interface` (?) instead of `$dis_interface`. And the `$wantsink` param looks a little too harsh, and I think `$display_sink` (?) would be more consistent with the other similar parameters.

Reply:
Yes, make sense.